packages/nimble-components/src/rich-text/specs/README.md

```diff
@@ -200,6 +332,68 @@ _CSS Classes and CSS Custom Properties that affect the component_
 
 _Note_: This initial component design serves as a starting point for implementation, and it may undergo changes once the visual design is completed.
 
+#### User mention element:
+
+This element serves as a container for the mapping elements and additionally functions as a configuration element. It allows the transmission of
+arbitrary values that are specific to the `@mention` users within the rich text components.
```

Comment:
Some examples of arbitrary values here or in the first place we mention `arbitrary values` will help understand the possible use cases. 

May be a good use case - Any icons we would like to prefix and customize the list?

Reply:
I added an example to the above point and explicitly mentioned that currently, we don't have any `arbitrary` values for this configuration element.